sql/localsql/schema/schema.sql

```diff
@@ -0,0 +1,82 @@
+PRAGMA user_version = 8;
```

Comment:
You're never expected to edit schema file by hand. You run `TestSchema` in `localsql` and `statesql` package. The test runs all the migrations starting from an empty db and captures the new schema, which it compares to the stored one. If there are schema differences, the test writes an updated schema file into `schema/schema.sql.updated`. If the diff shown by the test is acceptable, you do
```console
$ mv sql/schema/schema.sql{.updated,}
```
and then the tests passes. Basically, the so called "golden master" approach.

Reply:
Now thinking about this instead of doing this manually by executing a test and moving files around this is something that should happen on `make generate` in my opinion, since this is a generated file and just like go scale encoding and mocks should be updated on `make generate`. Then we also don't need the tests since we are already checking in our CI that `make generate` is a no-op or fail the build (even before tests and builds are executed). Maybe there is a quick and easy way to make this happen?

Reply:
`schema.sql` files are now handled by `make generate`, and consequently verified by `make test-generate` which is a part of the CI workflow